internal/config/config.go

```diff
@@ -198,6 +206,49 @@ func (c *Configuration) SanitizeAPIUI(ctx context.Context) (err error) {
 		log.Info(ctx, "Issuer DID not provided in configuration file")
 	}
 
+	err = c.sanitizeCredentialStatus(ctx, c.APIUI.ServerURL)
+	if err != nil {
+		log.Error(ctx, "error sanitizing credential status", "error", err)
+		return err
+	}
+	return nil
+}
+
+func (c *Configuration) sanitizeCredentialStatus(_ context.Context, host string) error {
+	if c.CredentialStatus.RHSMode != "" && c.CredentialStatus.RHSMode != onChain && c.CredentialStatus.RHSMode != offChain && c.CredentialStatus.RHSMode != none {
+		return fmt.Errorf("ISSUER_CREDENTIAL_STATUS_RHS_MODE value is not valid")
+	}
+
+	if c.CredentialStatus.RHSMode == none {
```

Comment:
should the case "" be here as well? if not we will return nil and may cause some some problems later on with empty vars

`if c.CredentialStatus.RHSMode == none || c.CredentialStatus.RHSMode == ""`